### PR TITLE
Quick fix for pr from SWAT team

### PR DESCRIPTION
**Pull request by oshmyheliuk**
### Description
Fix for the latest release.
https://jira.corp.magento.com/browse/MCLOUD-6156

### Associated documentation updates

https://github.com/magento/devdocs/pull/7462

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] Pull request introduces user-facing changes and includes meaningful updates for any required release notes and documentation changes
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

**Review by shiftedreality — DISMISSED** [review]

File: Model/Cache/InvalidateLogger.php [hunk 1]
```diff
@@ -20,7 +20,10 @@ class InvalidateLogger extends \Magento\Framework\Cache\InvalidateLogger
      */
     public function execute($invalidateInfo)
     {
-        $invalidateInfo['trace'] = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
+        if (is_array($invalidateInfo)) {
+            $invalidateInfo['trace'] = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
```
Comment (bbatsche) [inline Model/Cache/InvalidateLogger.php:24]: We're ok with not always having this info?

**Review by bbatsche — DISMISSED** [review]

**Review by shiftedreality — DISMISSED** [review]

File: Model/Cache/InvalidateLogger.php (hunk 1 above)
Comment (shiftedreality) [reply Model/Cache/InvalidateLogger.php:24]: This should be is something to fix for SWAT team next time

Comment (oshmyheliuk) [reply Model/Cache/InvalidateLogger.php:24]: @Alkar778 are we ok with this for now?

Comment (Alkar778) [reply Model/Cache/InvalidateLogger.php:24]: This debug backtrace is the main goal of this improvement. 
I will try to fix it in a couple of hours

**Review by bbatsche — APPROVED** [review]
There's a cleaner way to handle this I think, but it's not worth messing with at this time

**Review by shiftedreality — APPROVED** [review]

**Comment by andriyShevtsov** [comment]
QA approved

**Pull request merged**